docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -174,6 +174,17 @@ The `RefreshScope` is a bean in the context and has a public  `refreshAll()` met
 The `/refresh` endpoint exposes this functionality (over HTTP or JMX).
 To refresh an individual bean by name, there is also a `refresh(String)` method.
 
+To expose the `/refresh` endpoint you will need to add following configuration to your application.
```

Comment:
Change to "To expose the `/refresh` endpoint, you need to add following configuration to your application:"